### PR TITLE
Don't prevent a merge if cargo deny can't be built.

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

File: .buildbot.sh [hunk 1]
```diff
@@ -18,5 +18,7 @@ rustup default stable
 cargo test
 cargo test --release
 
-which cargo-deny | cargo install cargo-deny
-cargo-deny check license
+which cargo-deny | cargo install cargo-deny || true
+if [ "X`which cargo-deny`" != "X]; then
```
Comment (ptersilie) [inline .buildbot.sh]: Is this missing a closing `"` or is `"X` a special bracket type?

Comment (vext01) [inline .buildbot.sh]: Does this parse? Looks like there might be a missing double quote?

**Comment by ltratt** [comment]
Well spotted both! Fixed in https://github.com/softdevteam/grmtools/pull/171/commits/cbc2e4b51347914d1ff83b706e608789c7b8e1e3.

**Comment by vext01** [comment]
LGTM

**Comment by ptersilie** [comment]
Please squash.

**Comment by ltratt** [comment]
Squashed.

**Pull request merged**